chunky/src/java/se/llbit/chunky/block/MinecraftBlockProvider.java

```diff
@@ -1549,7 +1552,7 @@ public Block getBlockByTag(String namespacedName, Tag tag) {
       case "ice":
         return new MinecraftBlockTranslucent(name, Texture.ice);
       case "snow_block":
-        return new MinecraftBlock(name, Texture.snowBlock);
+        return new MinecraftBlockTranslucent(name, Texture.snowBlock);
```

Comment:
is snow actually translucent?

Reply:
In Minecraft, it isn't, but in real life, snow has subsurface scattering, so this change will allow the user to add subsurface scattering if they desire.